simcon_project/templates/feedback/view_feedback.html

```diff
@@ -40,7 +40,11 @@ <h2>Overall feedback:</h2>
         </div>
         {% for node in response_nodes %}
         <div class="row">
-            <h3> Response {{node.position_in_sequence}}</h3>
+            <h3> Response {{node.position_in_sequence}}
+            {% if node.custom_response != Null %}
+               (Custom)
```

Comment:
I think Custom is more of a coding implementation not exactly known to anybody but us... i would prefer a something visual to draw attention rather than an implementation detail

Reply:
I say instead of 
**selected choice:** "..."
do 
**custom response:** [contents of custom response]

Reply:
I honestly think **Typed Response** makes the most sense to an outside user